components/automate-ui/src/app/modules/team/team-details/team-details.component.ts

```diff
@@ -8,13 +8,23 @@ import { filter, map, takeUntil, distinctUntilChanged } from 'rxjs/operators';
 
 import { LayoutFacadeService, Sidebar } from 'app/entities/layout/layout.facade';
 import { ChefSorters } from 'app/helpers/auth/sorter';
+import { Regex } from 'app/helpers/auth/regex';
```

Comment:
PR bonus: Changes in this file are just re-ordering imports per Chef-convention/best practice.